apps/src/redux/instructions.js

```diff
@@ -30,7 +31,7 @@ const LOCALSTORAGE_OVERLAY_SEEN_FLAG = 'instructionsOverlaySeenOnce';
  *     longInstructions will both be set.
  * (5) CSF level with just long instructions
  */
-const instructionsInitialState = {
+export const InstructionsState = Record({
   noInstructionsWhenCollapsed: false,
```

Comment:
The `Record()` function produces a factory (essentially a new type) that will always have the keys given in this initial template and constructs with the default values given here.

```js
const ABRecord = Record({ a: 1, b: 2 })
const myRecord = new ABRecord({ b: 3 })
myRecord.equals({ a: 1, b: 3 });
```